arch/arm/adler32_neon_dotprod.c

```
@@ -0,0 +1,31 @@
+/* Copyright (C) 1995-2011, 2016 Mark Adler
+ * Copyright (C) 2017 ARM Holdings Inc.
+ * Copyright (C) 2025 Nathan Moinvaziri
+ * Authors:
+ *   Adenilson Cavalcanti <adenilson.cavalcanti@arm.com>
+ *   Adam Stylinski <kungfujesus06@gmail.com>
+ *   Nathan Moinvaziri <nathan@nathanm.com>
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#if defined(ARM_NEON) && defined(ARM_NEON_DOTPROD)
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)